zip-xxxx.rst

```diff
@@ -187,16 +207,16 @@ Transaction Format
   ``vOutputsSapling`` and MUST be ordered such that the proof at a given index corresponds
   to the ``OutputDescriptionV6`` at the same index.
 
-* The proofs aggregated in ``proofsOrchard``, and the elements of
+* The proofs aggregated in ``proofsOrchardZSA``, and the elements of
   ``vSpendAuthSigsOrchard``, each have a 1:1 correspondence to the elements of
   ``vActionsOrchard`` and MUST be ordered such that the proof or signature at a given
-  index corresponds to the ``OrchardAction`` at the same index.
+  index corresponds to the ``ZSAOrchardAction`` at the same index.
 
-* For coinbase transactions, the ``enableSpendsOrchard`` bit MUST be set to ``0``.
+* For coinbase transactions, the ``enableSpendsOrchard`` and the ``enableZSAs`` bits MUST be set to ``0``.
```

Comment:
```suggestion
* For coinbase transactions, the ``enableSpendsOrchard`` and ``enableZSAs`` bits MUST be set to ``0``.
```
Without the suggestion, this reads to me as "the ``enableSpendsOrchard`` bits and the ``enableZSAs`` bits" which is not intended, since they are each one bit.
Saying "the ``enableSpendsOrchard`` bit and the ``enableZSAs`` bit MUST both be set to ``0``" would also be correct.